pennylane/templates/subroutines/controlled_sequence.py

```diff
@@ -126,7 +131,7 @@ def map_wires(self, wire_map: dict):
 
     # pylint:disable=arguments-differ
     @staticmethod
-    def compute_decomposition(*_, base=None, control_wires=None, **__):
+    def compute_decomposition(*_, base=None, control_wires=None, lazy=False, **__):
```

Comment:
If you want it to print nicely, you should use `lazy=True`. If you want it to be differentiable, you should leave it as `lazy=False`, because otherwise you have a power operator and ML interfaces fail for gradients. I'm not sure it should be added to the hyperparameters. In this case, it's about *how you want the decomposition done*, rather than a description of the operator. But maybe the issue is that power operators should decompose differently and this isn't the place to fix it, let me look at it again.

Reply:
I misunderstood on first review, but now see that the default `lazy` value used to be `True` when decomposing, and this PR makes it `False`. lgtm!